modin/core/dataframe/pandas/dataframe/dataframe.py

```diff
@@ -2717,34 +2739,34 @@ def broadcast_apply_full_axis(
             keep_partitioning=keep_partitioning,
             apply_func_args=apply_func_args,
         )
-        kw = {"row_lengths": None, "column_widths": None}
+        new_dtypes = None
         if dtypes == "copy":
-            kw["dtypes"] = self._dtypes
+            new_dtypes = self._dtypes
         elif dtypes is not None:
             if new_columns is None:
                 (
                     new_columns,
-                    kw["column_widths"],
+                    new_column_widths,
```

Comment:
In some cases, you are overwriting variables that are passed in from outside. In other cases, you only do this when the variable is None. Let's get rid of this inconsistency.

Reply:
not actual